apps/dev/pages/server.js

```diff
@@ -11,11 +11,11 @@ export default function Page() {
     <Layout>
       <h1>Server Side Rendering</h1>
       <p>
-        This page uses the universal <strong>getSession()</strong> method in{" "}
-        <strong>getServerSideProps()</strong>.
+        This page uses the universal <strong>getServerSession()</strong> method
```

Comment:
`getServerSession` is not universal, it's server-only. Basically:

Before:
 - getSession: client, server

After:
 - getSession: client
 - getServerSession: server